conference/debug_panel.py

```diff
@@ -135,3 +136,15 @@ def debug_panel_invoice_export_csv(request):
     export_invoices_to_2018_tax_report_csv(response, start_date, end_date)
 
     return response
+
+
+@staff_member_required
+def debug_panel_invoice_export_accounting_json(request):
+    response = HttpResponse(content_type='application/json')
```

Comment:
This view can be simplified to using JsonResponse. (argubaly we could do CsvResponse as well, but JsonResponse is already builtin whereas CsvResponse would have to be implemented)